packages/chain-events/cosmos-chain-testing/start.sh

```diff
@@ -12,4 +12,4 @@ if [ "$1" = "--build" ]; then
     cd $SCRIPT_DIR
     docker build -t ${IMAGE_NAME} .
 fi
-docker run -p 5050:5050 -e PORT=${PORT} --name ${IMAGE_NAME} ${IMAGE_NAME}
\ No newline at end of file
+docker run -p 5050:5050 -e PORT=${PORT} --name ${IMAGE_NAME} ${IMAGE_NAME} & START_PID=$!
```

Comment:
adding this was key to running the chain in the background so the next CI step could run. But let me know if there is a better way. The way it is now, it takes a long time to build and run the devnet from scratch.

@NakulManchanda

Reply:
didn't understand this, but rest looks right!

Reply:
yeah we can start in background, if we don't want to wait for image to download everytime!
cool, thats fine, thanks!

Reply:
- this helps locally only, script doesn't get called via CI.yml at all
- other option is use `-d` detached mode
```
docker run -p 5050:5050 -e PORT=${PORT} --name ${IMAGE_NAME} -d
```

We kept it this way for local to see streaming logs in terminal during startup to spot errors if any, we can always stream like this later
```
docker logs -f ${IMAGE_NAME}
```